docs/analyzer/index.md

```diff
@@ -113,40 +113,4 @@ For a list of the current supported entities:
 
 ## API reference
 
-### Calling Presidio Analyzer via HTTP
-
-`/analyze`
-
-Analyzes a text. Method: `POST`
-
-Parameters
-
-| Name | Type | Optional | Description|
-| --- | --- | ---| ---|
-| text|string|no|the text to analyze|
-| language|string|no|two characters for the desired language [ISO_639-1 language code](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes). E.g en, de|
-| correlation_id|string|yes|a correlation id to append to headers and traces|
-| score_threshold|float|yes|the the minimal detection score threshold|
-| entities|string[]|yes|a list of entities to analyze|
-| return_decision_process|bool|yes|whether to include analysis explanation in the response |
-
-`/recognizers`
-
-Returns a list of supported recognizers.
-Method: `GET`
-
-Parameters
-
-| Name | Type | Optional | Description|
-| --- | --- | ---| ---|
-| language|string|yes|2 characters of the desired language code. e.g., en, de. |
-
-`/supportedentities`
-
-Returns a list of supported entities. Method: `GET`
-
-Parameters
-
-| Name | Type | Optional | Description|
-| --- | --- | ---| ---|
-| language|string|yes|2 characters of the desired language code. e.g., en, de. |
+Follow the [API Spec](https://microsoft.github.io/presidio/api-docs/api-docs.html#tag/Analyzer) for Presidio API reference details
```

Comment:
Maybe also reference the Python API reference? (docs/api/analyzer_python.md)

Reply:
@shiranr I guess we want the same thing for the anonymizer doc

Reply:
added